app/experimenter/nimbus-ui/package.json

```diff
@@ -18,8 +18,16 @@
   },
   "jest": {
     "collectCoverageFrom": [
-      "!src/**/*.stories.*"
-    ]
+      "src/**/*",
+      "!src/**/*.stories.*",
+      "!src/lib/mocks.tsx",
+      "!src/lib/test-utils.tsx"
```

Comment:
Oh my glob, you could, like, totally simply these two lines to `"!src/lib/{mocks,test-utils}.tsx"`